site/content/contextual-data-platform/tutorials/evaluate-locally.md

```
@@ -0,0 +1,625 @@
+  --namespace minio --timeout=120s
+```
+
+Confirm that MinIO is running and the bucket job completed:
+
+```sh
+kubectl get pods --namespace minio
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)